site/docs/plugins/i18n.md

```diff
@@ -1,105 +1,742 @@
-# Internationalization Support (`i18n`)
+# Internationalization (`i18n`)
 
-When you want to target a multilanguage audience with your bot, you will need internationalization.
-The grammy i18n plugin helps you with that.
+The internationalization plugin helps you write a bot that can speak multiple languages.
 
-## Usage
+## Internationalization Explained
+
+> This part explains what internationalization is, why it's needed, what is complicated about it, how it relates to localization, and why you need a plugin for all of this.
+> If you already know these things, scroll down to [get started](#getting-started).
+
+First, internationalization is a very long word, so people like to write the first letter (i) and the last letter (n).
+They then count all remaining letters (nternationalizatio, 18 letters) and put this number between i and the n, so they end up with _i18n_.
+Don't ask us why.
+So i18n is just a weird abbreviation of the word internationalization.
+
+The same is done to localization, which turns into _l10n_.
+
+### What Is Localization?
+
+Localization means creating a bot that can speak multiple languages.
+It should automatically adjust its language to the language of the user.
+
+There are more things to localize than the language.
+You can also account for cultural differences or other standards, such as the date and time formats.
+Here are a few more examples of things that are represented differently across the globe:
+
+1. Dates
+2. Times
+3. Numbers
+4. Units
+5. Pluralization
+6. Genders
+7. Hyphenation
+8. Capitalization
+9. Alignment
+10. Symbols and icons
+11. Sorting
+
+… and [a lot more](https://youtu.be/0j74jcxSunY).
+
+All of these things collectively define the _locale_ of a user.
+Locales often get two-letter codes, such as `en` for English, `de` for German, and so on.
+
+### What Is Internationalization?
+
+In a nutshell, doing internationalization means writing code that can adjust to a user's locale.
+In other words, internationalization is what enables localization (see [above](#what-is-localization)).
+This means that while your bot fundamentally works the same way for everybody, the exact messages it sends vary from user to user, so the bot can speak different languages.
+
+You are doing internationalization if you don't hard-code the texts your bot sends but instead read them from a file dynamically.
+If you don't hard-code how dates and times are represented but instead use a library that adjusts these values to different standards, you are doing internationalization.
```

Comment:
```suggestion
If you don't hard-code how dates and times are represented but instead use a library that adjusts these values according to different standards, you are doing internationalization.
```

Reply:
Hi, the comma before and isn't correct; or am i wrong